extensions/resource-deployment/package.json

```diff
@@ -255,7 +255,7 @@
               },
               {
                 "name": "azdata-old",
```

Comment:
remind me what azdata-old refers to?

Reply:
It's the "old" tool definition, which is basically for built in extensions to check the azdata version since the azdata extension isn't necessarily going to be installed (just BDC currently)

For other extensions (such as arc) we can use the azdata extension so the tool for that is just "azdata" and it relies on that extension prompting people to update to the latest before running commands.

azdata-old will be removed once BDC is moved into its own extension.

Reply:
Thanks